deployment/common/changeset/link_transfer_timelock.go

```diff
@@ -0,0 +1,144 @@
+package changeset
+
+import (
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/types"
+)
+
+type LinkTransfer struct {
+	To    common.Address
+	Value *big.Int
+}
+type LinkTransferTimelockRequest struct {
```

Comment:
nit: would name it XXConfig

Reply:
renamed here too: https://github.com/smartcontractkit/chainlink/pull/15512/commits/ba6b57daeb9509733ba0e4adc4ff4229b7c1ac82